docs/installation/binary_installation.md

```diff
@@ -0,0 +1,58 @@
+# Binary Installation
+
+Visualization packages with CARET can also be installed in binary.
+
+<prettier-ignore-start>
+!!! Note
+    Only the visualization packages can be installed by binary installation.
+    The binary does not include measurement packages.
+<prettier-ignore-end>
+
+CARET is confirmed to run on the platforms shown in the following table with supported version.
+
+| dependent platform | supported version |
+| ------------------ | ----------------- |
+| ROS                | Humble            |
+| Ubuntu             | 22.04             |
+| LTTng              | stable-2.13       |
+| Linux Kernel       | 5.15.x            |
+| Python3            | 3.10.x            |
+
+## Installation of required packages
+
+Update your apt repository caches.
+
+```
+sudo apt update
+```
+
+The following packages are required for the CARET binary packages. These dependencies are not automatically installed with the binary installation.
+
+```
+sudo apt install python3-bt2
+python3 -m pip install -U \
+  'pandas>=1.4.0' \
+  bokeh \
+  jupyterlab \
+  multimethod
+```
+
+## Install CARET packages
+
+Visualization packages for CARET install: caret_analyze, caret_analyze_cpp_impl, ros2caret, caret_msgs.
+
+```
+sudo apt install ros-humble-caret-analyze
+sudo apt install ros-humble-caret-analyze-cpp-impl
+sudo apt install ros-humble-ros2caret
+sudo apt install ros-humble-caret-msgs
+```
+
+Please open Jupyter Lab.
+
+```
+export PATH=$PATH:~/.local/bin
+jupyter-lab
+```
+
+Recommend trying CARET functionality by following the tutorial.
```

Comment:
How about running `ros2 caret version` to make sure it was installed successfully?

Reply:
I thought it too.
However,  current CARET binary fails `ros2 caret version` command.
It takes a month to fix it. Is that okay?

Reply:
Fixed in https://github.com/tier4/caret_doc/pull/335/commits/5005d293b1a292daea875397db3ae5a7bf2e3d4a.

Reply:
Indeed, the version command cannot be specified in the current release of ros2caret.
A fix for this bug will be released in the near future.
You may want to mention in the documentation that it is temporarily unavailable due to a bug.
Wecan confirm that the installation was successful by using the alternative method described below.
```
root@f2fd836ae725:/# source /opt/ros/humble/setup.bash
root@f2fd836ae725:/# ros2 pkg list | grep caret
caret_analyze
caret_analyze_cpp_impl
caret_msgs
ros2caret
```

Reply:
Thanks, fixed in https://github.com/tier4/caret_doc/pull/335/commits/be4b16f78f3009164e7926747930717f2ab0e687

Reply:
@h-suzuki-isp 
It looks good.
However, `root@f2fd836ae725` is specific user information used in the experiment and does not need to be included in the document. Please change the description to be similar to other shell descriptions.

Reply:
Note that this description must be deleted in the future for this writing style.